src/xenia/kernel/xam/xam_enum.cc

```diff
@@ -41,27 +41,8 @@ uint32_t xeXamEnumerate(uint32_t handle, uint32_t flags, lpvoid_t buffer_ptr,
   } else if (!buffer_ptr) {
     result = X_ERROR_INVALID_PARAMETER;
   } else {
-    size_t needed_buffer_size = e->item_size() * e->items_per_enumerate();
-
-    uint32_t actual_buffer_size = buffer_size;
-    if (buffer_size == e->items_per_enumerate()) {
-      actual_buffer_size = static_cast<uint32_t>(needed_buffer_size);
-      // Known culprits:
-      //   Final Fight: Double Impact (saves)
```

Comment:
Are saves "content" here that your enumerator would correct Final Fight's behavior? I don't recall.